magclassic/automated_emails.py

```diff
@@ -1,19 +1,25 @@
 from magclassic import *
 
-# With "MAGFest Laboratories" as a new event, we find that the phrasing "Staffing again"
-# doesn't differentiate this event, so we remove it.
-AutomatedEmail.instances['Want to staff {}?'.format(c.EVENT_NAME)] = AutomatedEmail.instances.pop('Want to staff {} again?'.format(c.EVENT_NAME))
+# With "MAGFest Laboratories" as a new event, we find that the phrasing "Want to staff again?"
+# doesn't differentiate this event, so we change it to just plain "Want to staff?"
+if 'volunteer_again_inquiry' in AutomatedEmail.instances:
```

Comment:
It is! I mentioned this on Slack already but for the benefit of anyone else reading:
The idea behind removing 'again' for this email subject was to brand MAG Labs as a "new" event. We've had a fall event three years running, but it was renamed every year until we settled on MAG Labs. We didn't want volunteers to be confused when we said "Want to staff MAGFest Laboratories again?" -- most people would be like, "when did I staff it the FIRST time?" (because they actually staffed "MAG Classic," not "MAG Labs") Now that the next fall event will also be "MAGFest Laboratories," we DO want these email subjects to say 'again' -- it's the second event of its kind.

For the same reason, this line only should be removed in THIS repo. The version you have here is perfect for MAGWest, because it is a new event.